{"name": "测试用例描述：错误的账号登录失败", "status": "passed", "description": "测试描述：错误的账号登录失败", "steps": [{"name": "输入错误的账号", "status": "passed", "attachments": [{"name": "错误的账号", "source": "952238bd-ac7c-4fa1-b175-95b539d866ca-attachment.attach"}], "start": 1651987995557, "stop": 1651987995557}], "attachments": [{"name": "截图", "source": "e5a2d2ba-9b85-4736-af6b-37fb1cf2e1ea-attachment.png", "type": "image/png"}, {"name": "截图", "source": "a8573073-9513-4232-9de2-75edd7aee2c6-attachment.png", "type": "image/png"}], "start": 1651987995557, "stop": 1651987999269, "uuid": "f2b803fe-0631-4556-80d0-f94df3d96298", "historyId": "07ccc934c01f5e078d85f2f38ccbb5fd", "testCaseId": "07642125497ce5d4c888927c76cac843", "fullName": "test_two.Test#test_21", "labels": [{"name": "epic", "value": "xxx测试"}, {"name": "severity", "value": "critical"}, {"name": "story", "value": "登录"}, {"name": "feature", "value": "登录模块"}, {"name": "suite", "value": "test_two"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "18340-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_two"}]}